CBLClient/Apps/CBLTestServer-Dotnet/TestServer/FileLoggingMehtod.cs

```diff
@@ -156,6 +157,42 @@ public class FileLoggingMehtod
 
         }
 
+        public static void GetLogsInZip([NotNull] NameValueCollection args,
+                                [NotNull] IReadOnlyDictionary<string, object> postBody,
+                                [NotNull] HttpListenerResponse response)
+        {
+            var path = Database.Log.File.Config?.Directory;
```

Comment:
We create separate logs for each test, so we need to pass directory for this purpose. Change this code to accept the path string provided by user.

I've used below change for my testing purpose

var path = postBody["path"].ToString();

Reply:
You mean that by the time you call this method, the current log file directory will no longer be valid?

Reply:
for most of the cases what you say is true, but I remember we have test cases where we have created more than 1 db. It's not giving any issue for now. Will comeback to it later if it gives issue for that case